### ISSUE TITLE
different keyboard layouts, US layout is broadly used in EU

### ISSUE THREAD

**Issue by rudenick1** [egaga/openvim#70]

**Comment by LightningAA** [comment]
I've had issues trying to use the dvorak keyboard layout

**Comment by egaga** [comment]
Yup, not supported. Outside of the scope of low-burning hobby project.

But good to know that there is demand, thanks.

**Comment by unelma** [comment]
#85 has partial support. Somebody just needs to make layout configuration for US keyboard.
